src/screen/ChooseRoomScreen.js

```diff
@@ -0,0 +1,49 @@
+import React, { useState, useEffect } from 'react';
+import { Button, Input } from 'react-native-elements';
+import { useTranslation } from 'react-i18next';
+import { connect } from 'react-redux';
+import WebSocket from '../api/socket';
+import Block from '../components/Block';
+import { Navigation } from '../constants/Navigation';
+
+const ChooseRoomScreen = ({
+  navigation,
+  userName,
+}) => {
+  useEffect(() => {
+    WebSocket.runSocket();
+  }, []);
+
+  const [name, setRoomName] = useState();
```

Comment:
`  const [name, setRoomName] = useState();`
I think if you use `setRoomName` then the variable should be called `roomName`